lamp-stack-ubuntu/README.md

```diff
@@ -0,0 +1,6 @@
+# Lamp stack on Ubuntu VM
```

Comment:
@251744647 We try to minimize templates with same functionality. In that case, I suggest editing the existing template with these enhancements.